### DISCUSSION TITLE
Min time

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#9329] [Help]
How can I add min time to time picker?

**Comment by leandrocfe** [comment]
I'm on mobile because my laptop is broken 🤒

Not sure if there is an option like "minTime", but I think you can inject an attribute like

`->native(true)->extraInputAttributes(['min' => '08:00'])`

**Reply by ArtMin96** [reply]
Thanks @leandrocfe. I don't want to use the native time input, besides, it should display the min time message when selecting the time, not when submitting the form.
